src/components/Story/Story.tsx

```diff
@@ -15,15 +18,46 @@ import {
   Pagination,
 } from 'swiper/modules';
 
-import { SettingsButton, Slide, Slider } from '@/components/Story/Story.styled';
+import Exitbox from '@/components/Exitbox';
+import {
+  FullscreenButton,
+  ReturnToMainPage,
+  SettingsButton,
+  Slide,
+  Slider,
+} from '@/components/Story/Story.styled';
 import StoryPage from '@/components/StoryPage';
-import { Toolbox } from '@/components/Toolbox';
+import Toolbox from '@/components/Toolbox';
 
 import type { Story as StoryType } from './Story.types';
 
 export const Story: StoryType = ({ storyContent }) => {
   const swiperRef = useRef(null);
   const [settingsVisibility, setSettingsVisibility] = useState<boolean>(false);
+  const [exitVisibility, setExitVisibility] = useState<boolean>(false);
+  const [fullscreen, setFullscreen] = useState<boolean>();
+
```

Comment:
wrzuć jako zmienne `document.documentElemen...` i `document.exitFullscreen` a potem zamiast pisać `if` / `else` to daj ternary

`!document.fullscreenElement ? zmiennaA : zmiennaB`

Reply:
nie będzie tak boleć :P

Reply:
Zrobione